### DISCUSSION TITLE
Mengubah pickel ke csv/excel

### DISCUSSION THREAD

**Discussion by ChristoNindyo** [teamalgoritma/community#166] [Q&A] [answered]
Kalau misalkan mengubah pickle ke csv/excel dengan format yang sama bisa engga ya ? Bagaimana caranya ?

**Comment by nisa-basalamah** [comment, marked as answer]
Halo @ChristoNindyo ,

Hal ini tergantung pada jenis/format data yang bapak simpan pada file pickle tersebut. Jika data yang disimpan dalam file pickle merupakan sebuah dataframe, tentunya bisa untuk diubah ke csv/excel. Berikut langkah yang dapat dikerjakan untuk melakukan proses ini,

1. Baca file pickle menggunakan `pd.read_pickle()` (object: DataFrame)
2. Simpan object tersebut menggunakan `df.to_csv()` atau `df.to_excel()`

Sebagai informasi tambahan, bapak dapat menggunakan referensi berikut [How to import CSV file with multi-level columns (Python Basics)](https://medium.com/@gotashirato/how-to-import-csv-file-with-multi-level-columns-python-basics-and-a-question-b67cbbbc174d) untuk meng-handle file csv yang berisi data multi-index.

Semoga bermanfaat :)
